templates/games/4form.html

```diff
@@ -33,14 +33,14 @@ <h3>4tos</h3>
         <div class="input-group">               
             <span class="input-group-text" id="q12" name="teamA" style="width: 30%;"></span>
             <input type="hidden" id='gameId' name="gameId" value="q12" />
-            <input type="number" min="0" class="form-control" name="resA" id="q12" placeholder="W7" required>
-            <input type="number" min="0" class="form-control" name="resB" id="q12" placeholder="W8" required>
+            <input type="number" min="0" class="form-control" name="resA" id="q12" style="width: 10%;"placeholder="W7" required>
+            <input type="number" min="0" class="form-control" name="resB" id="q12" style="width: 10%;"placeholder="W8" required>
             <span class="input-group-text" id="q12" name="teamB" style="width: 30%;"></span>
             <div class="col-sm">
                 <select id="q12-select" name="winner" style="visibility: hidden;"></select>
             </div>
         </div>
-<div class="col-md-12">
+<div class="col-md-12" style="margin : 10px auto;">
```

Comment:
This add an space between scores and button and make button appear at the center